pkg/autoscaler/config/config.go

```diff
@@ -131,16 +135,24 @@ func validate(lc *autoscalerconfig.Config) (*autoscalerconfig.Config, error) {
 		return nil, fmt.Errorf("target-burst-capacity must be either non-negative or -1 (for unlimited), was: %f", lc.TargetBurstCapacity)
 	}
 
-	if lc.ContainerConcurrencyTargetFraction <= 0 || lc.ContainerConcurrencyTargetFraction > 1 {
+	if lc.ContainerConcurrencyTargetFraction < 0 || lc.ContainerConcurrencyTargetFraction > 1 {
 		return nil, fmt.Errorf("container-concurrency-target-percentage = %f is outside of valid range of (0, 100]", lc.ContainerConcurrencyTargetFraction)
 	}
 
-	if x := lc.ContainerConcurrencyTargetFraction * lc.ContainerConcurrencyTargetDefault; x < autoscaling.TargetMin {
+	if lc.TargetUtilization <= 0 || lc.TargetUtilization > 1 {
+		return nil, fmt.Errorf("target-utilization = %f is outside of valid range of (0, 100]", lc.TargetUtilization)
+	}
+
+	if x := lc.ContainerConcurrencyTargetFraction * lc.ContainerConcurrencyTargetDefault; lc.ContainerConcurrencyTargetFraction != 0 && x < autoscaling.TargetMin {
```

Comment:
if `ContainerConcurrencyTargetFraction` is getting deprecated, should we stop using it and just set the new field `TargetUtilization` to its value? makes removing it in the future easier

Reply:
I'd remove it in a follow-up PR.

Reply:
I think the change would be simpler if we dropped usage of `ContainerConcurrencyTargetFraction` in favour of `TargetUtilization` in the config go type